src/plugins/intel_gna/src/optimizer/gna_pass_manager.cpp

```diff
@@ -2436,6 +2467,90 @@ void TransposeWeightsFromNCHWToNHWCPass::run() {
     }
 }
 
+void FuseFullyConnectedWithEltwisePass::run() {
+    // This legacy pass removes the Eltwise (only if it performs SUM op) from between FC and Any.
+    // The Const layer blob data is added to biases blob data of FC layer.
+    // Finally Const is also removed
+    // Reshape (if exists) is moved after Any to keep output shape unchanged.
+    //
+    // This operation can be illustrated as follows:
+    //
+    // --
+    //  Original:                  Result:           Removed:
+    //
+    //    FC                         FC              (Eltwise)
+    //    |                          |               (Const)
+    //  Reshape                   Reshape
+    // (optional)  Const    (if exists in Original)
+    //    |       /                  |
+    //  Eltwise(sum)                Any
+    //    |
+    //  Any (e.g. ReLU)
+    //--
+    //
+    // NOTE: This pass is implemented to prevent unnecessary roundtrip to memory (additional layer).
+    // It can be fully removed if corresponding LPT transformation is already implemented.
+
+    OV_ITT_SCOPED_TASK(itt::domains::GNA_LT, "FuseFullyConnectedWithEltwisePass");
+    for (auto& l : *pLayers) {
+        if (!LayerInfo(l).isFullyConnected()) {
+            continue;
+        }
+        auto& fc_layer = l;
```

Comment:
`l` looks strange
![image](https://user-images.githubusercontent.com/106798706/216015975-77a7d0f6-afdc-4ded-911e-c156113c1338.png)
Can we change it to
```
auto& fc_layer = layer;
```